test_util/cluster.py

```diff
@@ -372,6 +368,11 @@ def wait_for_mesos_metric(cluster, host, key, value):
             add_config_path=add_config_path,
             rexray_config_preset=cluster.rexray_config_preset,
         )
+
+        # Generate node upgrade script
+        output = installer.generate_node_upgrade_script()
+        upgrade_script_path = output.decode('utf-8').strip('\n').split("Node upgrade script URL: ", 1)[1]
```

Comment:
The unit test would capture it being in the last line, I can use the same thing here.